code/modules/mining/mint.dm

```diff
@@ -184,6 +194,14 @@
 					newCoins++
 					src.updateUsrDialog()
 					sleep(5);
+			if("mime")
+				while(amt_mime > 0 && coinsToProduce > 0)
+					create_coins(/obj/item/weapon/coin/mime)
+					amt_mime -= 20
+					coinsToProduce--
+					newCoins++
+					src.updateUsrDialog()
+					sleep(5);

```

Comment:
Wait, how is it bad? That is literally identical to all other ones before/after it, it should be fine.